include/vcpkg/registries.h

```diff
@@ -196,4 +211,7 @@ namespace vcpkg
     private:
         VersionDbEntryDeserializer underlying;
     };
+
+    // exposed for testing
```

Comment:
@ras0219-msft This is what I meant about "most details are public to someone" on Thursday. This thing is public, and tested like it is public.

```suggestion
```